### ISSUE TITLE
If chat is not properly setup, disable it

### ISSUE THREAD

**Issue by kordianbruck** [TCA-Team/TumCampusApp#338]
```
java.lang.NullPointerException: Attempt to invoke virtual method 'int de.tum.in.tumcampus.models.ChatMember.getId()' on a null object reference
	at de.tum.in.tumcampus.models.ChatVerification.<init>(ChatVerification.java:22)
	at de.tum.in.tumcampus.activities.ChatRoomsActivity.createOrJoinChatRoom(ChatRoomsActivity.java:205)
	at de.tum.in.tumcampus.activities.ChatRoomsActivity.onItemClick(ChatRoomsActivity.java:291)
```

We are still getting a lot of the above exceptions. We need to handle those events and disable the chat when the setup is not correct. (I.e.: PK creation failed, did not activate the token and so on)

**Comment by kordianbruck** [comment]
This is still an issue and we need to fix this asap. Leads to FC of the App.

**Comment by kordianbruck** [comment]
@pfent any other ideas on how to fix this issue? I don't have any idea on how it happens that the chat member is not set.

**Comment by pfent** [comment]
Oh wow, I wanted to reproduce this and ran into multiple issues.
Startup Wizzard: Ignore Email -> click on Next -> App crashes
Next startup, the Chat thinks, it is already enabled and doesn't show any new prompt to enable the chat.
Chat is still accessible from side menu

When clicking on the Chat from the side menu, the app crashes (probably the cause of #339)

**Comment by pfent** [comment]
Also we currently only check, if the user has ticked the "enable Chat" checkbox to show the Drawer item for the Chats. We should probably rework that, especially for older not probably setup users.

Probably introduce a new setting, if the chat is properly confirmed

**Comment by kordianbruck** [comment]
Should also be fixed, now that #339 is fixed
